index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "moleculer-apollo-server" {
+	import { ServiceSchema } from "moleculer";
+	import { Config } from "apollo-server-core";
+	export {
+		GraphQLUpload,
+		GraphQLExtension,
+		gql,
+		ApolloError,
+		toApolloError,
+		SyntaxError,
+		ValidationError,
+		AuthenticationError,
+		ForbiddenError,
+		UserInputError,
+		defaultPlaygroundOptions,
+	} from "apollo-server-core";
+
+	export * from "graphql-tools";
+
+	export interface ApolloServerOptions {
+		path: string;
+		disableHealthCheck: boolean;
+		onHealthCheck: () => {};
+	}
+
+	export class ApolloServer {
+		createGraphQLServerOptions(req: any, res: any): Promise<any>;
+		createHandler(options: ApolloServerOptions): void;
+		supportsUploads(): boolean;
+		supportsSubscriptions(): boolean;
+	}
+
+	export interface GraphQLNodeResolver {
+		action: string;
+		rootParams?: {
+			[key: string]: string;
+		};
+		dataloader?: boolean;
+	}
+
+	export interface GraphQLTypeResolver {
+		[key: string]: {
+			[key: string]: GraphQLNodeResolver;
+		};
+	}
+
+	export interface ApolloServiceOptions {
```

Comment:
This is missing the `schemaDirectives` property as well as the `createAction` property.  I'm unsure what the `createAction` property is supposed to do, but it appears to be `boolean` from looking at the moleculer-apollo-server source.

[Schema directives](https://www.apollographql.com/docs/apollo-server/schema/directives/) are passed straight through to `makeExecutableSchema`.

Reply:
This appears to be correct:
`schemaDirectives: { [key: string]: typeof SchemaDirectiveVisitor};`

with `SchemaDirectiveVisitor` coming from `graphql-tools`.

Reply:
I've checked in `moleculer-apollo-server` source. Should define `schemaDirectives` 👍